conformance_tests.yaml

```diff
@@ -3313,3 +3313,13 @@
     Test that optional number output is returned as 0, not null
   tags: [ required, inline_javascript, command_line_tool ]
   id: 253
+
+- tool: tests/arg-empty-prefix-separate-false.cwl
+  job: tests/empty.json
+  output: {}
```

Comment:
What is the expected output?

Reply:
The idea was to run command `echo non_existing_app` if everything works fine, and run `non_existing_app` if executor has a bug, so basically just relying on exit code as a check. Added a redirect to stdout so the output size and checksum are also validated.